CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to the Glean Annotations Repository
+
+Thank you for your interest in contributing to the Glean Annotations. This
+document tries to codify some best practices for contribution to this
+repository.
+
+## Participation guidelines
+
+All communication (on Slack and Matrix, on mailing lists, and in pull requests and issues) is expected to follow the [Mozilla Community Participation Guidelines](https://www.mozilla.org/about/governance/policies/participation/).
+For more information, see the [code of conduct document](./CODE_OF_CONDUCT.md)
+in the root of this repository.
+
+## Creating new annotations
+
+There are two recommended workflows for adding new annotations to this repository:
+
+1. Via the Metric's commentary section in the [Glean Dictionary](https://dictionary.protosaur.dev). This
+   will either create a new markdown document for you to put information in or allow you to modify the
+   existing one. When you're happy with your changes, convert your changes into a pull request. After
+   it is merged, your annotations should appear in the Glean Dictionary within a day.
```

Comment:
I'd be interested in talking more about this workflow. It should be possible for a contributor or contractor outside the mozilla github org to make a PR, but would it be safe to make this repository writeable by the whole mozilla github org such that employees could directly commit their changes rather than making a PR?

Are we expecting that PRs are reviewed? Who's responsible for reviewing and merging?

Reply:
This is a good question. I believe there was some suggestion in the product instrumentation proposal that DS would act as the reviewer/curator of this repository. I'd personally be happy to help review things, at least to get things started. I don't think we know at this point what kind of issues are likely to come up with people's contributions: we need more data. But I do think peer review is almost never a bad thing, as long as people don't have to wait for too long for it.

Maybe for now we could just go for something vague, like: "When you're happy with your changes, convert your changes into a pull request and someone from the Data organization will review it".

Reply:
Tried to be a bit more explicit about this.